contracts/interfaces/IUniswapV2Factory.sol

```diff
@@ -0,0 +1,9 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity 0.8.15;
+pragma abicoder v1;
+
+
+interface IUniswapV2Factory {
+    function INIT_CODE_PAIR_HASH() external view returns (bytes32);
```

Comment:
Good case, but there is no this method in official implementation which many projects inherit 
[UniswapV2Factory](https://github.com/Uniswap/v2-core/blob/v1.0.1/contracts/UniswapV2Factory.sol)
[IUniswapV2Factory](https://github.com/Uniswap/v2-core/blob/v1.0.1/contracts/interfaces/IUniswapV2Factory.sol)

Reply:
I am agree. Just added fix